Blockzilla/BrowserViewController.swift

```diff
@@ -69,8 +69,9 @@ class BrowserViewController: UIViewController {
     private var shouldEnsureBrowsingMode = false
     private var initialUrl: URL?
     
-    private let userDefaultsTrackersBlockedKey = "lifetimeTrackersBlocked"
-    private let userDefaultsShareTrackerStatsKey = "shareTrackerStats"
+    static let userDefaultsTrackersBlockedKey = "lifetimeTrackersBlocked"
+    static let userDefaultsShareTrackerStatsKeyOLD = "shareTrackerStats"
+    static let userDefaultsShareTrackerStatsKeyNEW = "shareTrackerStatsNew"
```

Comment:
You may not need this given my comment above.

Reply:
The old and new was to maintain the share button for users who have it. If we don't want that anymore then it simplifies the logic a bit.

Reply:
Uh, yeah, you are right, let's not shut off a feature for folks who have it. Sorry!